tests/test_preprocessing.py

```diff
@@ -255,13 +255,13 @@ def check_scaling_features(use_feature_hashing=False, use_scaling=False):
 
     # these are the expected values of the f-measures, sorted
     if not use_feature_hashing:
-        expected_fmeasures = ([0.77319587628865982, 0.78640776699029125] if
+        expected_fmeasures = ([0.55276381909547745, 0.55721393034825872] if
                               not use_scaling else
-                              [0.94930875576036866, 0.93989071038251359])
+                              [0.65217391304347827, 0.70370370370370372])
```

Comment:
Over the next few days, I am going to try and nail down what caused these differences. From what I can tell right now, It could be due to differences in:

- `make_classification_data`: perhaps the data being generated is actually different now?
- `SGDClassifier` - may be they changed something in the implementation?
- `FeatureHasher` - may be they changed something in the implementation?